azurerm/data_source_subnet.go

```diff
@@ -63,6 +63,11 @@ func dataSourceArmSubnet() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+
+			"disable_private_link_endpoint_network_policy_enforcement": {
```

Comment:
i presume this will be changed/overwritten from the service pr?

Reply:
Nope, they are side by side, there are two settings... I have updated the name to match the service.